src/java/org/apache/cassandra/replication/MutationTrackingService.java

```diff
@@ -88,9 +95,12 @@ public class MutationTrackingService
     /**
      * Split ranges into this many shards.
      * <p>
+     * REVIEW: Reset back to 1 because for transfers, replicas need to know each others' shards, since transfers are
+     * sliced to fit within shards. Can we achieve sharding via split range ownership, instead of it being local-only?
+     * <p>
```

Comment:
This was added as an aid to perf testing. I think reverting it to one is fine, especially since there's a functional reason here, but we should open a jira to revisit it